nextcord/ext/menus/menu_pages.py

```diff
@@ -167,14 +167,17 @@ class MenuPaginationButton(nextcord.ui.Button['MenuPaginationButton']):
     A custom button for pagination that will be disabled when unavailable.
     """
 
-    def __init__(self, style: nextcord.ButtonStyle, emoji: EmojiType):
-        super().__init__(style=style, emoji=emoji)
-        self._emoji = _cast_emoji(emoji)
+    def __init__(self, emoji: Optional[EmojiType] = None, **kwargs):
```

Comment:
maybe use Missing instead

Reply:
`Button` itself uses `None`, so I figure it's probably best to use the default there unless nextcord changes it.

https://github.com/nextcord/nextcord/blob/8711800697cbf9057ac3b88150e3dad7566c89bd/nextcord/ui/button.py#L95

It may also be worth considering leaving `emoji` as part of `kwargs` and just letting nextcord decide the default value.